string topic

float64 measured_latency
float64 measured_droprate
